### DISCUSSION TITLE
The nvim-tree background is not transparent when all other backgrounds are.

### DISCUSSION THREAD

**Discussion by ivandimitrov8080** [nvim-tree/nvim-tree.lua#1860] [Q&A]
Here's a screenshot showing what I'm talking about:

![image](https://user-images.githubusercontent.com/80976541/209473901-aca2e0d8-85d4-4186-9aa4-33b3a420657c.png)

You can see my config there. It's the default config. Is this a bug or do I need to change something to make it transparent?

**Comment by alex-courtis** [comment]
The background is coloured via `NvimTreeNormal` and `NvimTreeNormalNC` which both default to standard `Normal` highlight.

Please
```vim
:hi Normal
:hi NormalNC
:hi NvimTreeNormal
:hi NvimTreeNormalNC
```
